bigbluebutton-html5/imports/startup/mapToAcl.js

```diff
@@ -0,0 +1,39 @@
+import Acl from '/imports/startup/acl';
+import { Meteor } from 'meteor/meteor';
+import Logger from '/imports/startup/server/logger';
+
+const emptyCollection = new Mongo.Collection("emptyCollection");
```

Comment:
:thinking: Why do you find `this.ready()` confusing?
According by this post thread (https://forums.meteor.com/t/this-ready-in-a-publication/1681/4) you can return an empty array.